packages/app-admin-ui/index.js

```diff
@@ -52,6 +54,15 @@ class AdminUIApp {
     };
   }
 
+  isAccessAllowed(req) {
+    return (
+      req.user &&
+      this._isAccessAllowed({ authentication: { item: req.user, listKey: req.authedListKey } }) &&
+      req.session.audiences &&
+      req.session.audiences.includes('admin')
```

Comment:
This is potentially redundant now? @timleslie this was part of your initial experimental audiences work - is it useful to keep?

Reply:
Um, I'd prefer to keep it for now and either get it all to work in the future or tear it all out in one big hit if we decide its redundant.